etl/steps/data/garden/missing_data/2024-03-14/children_out_of_school.py

```diff
@@ -0,0 +1,120 @@
+"""Load a meadow dataset and create a garden dataset."""
+
+import numpy as np
+import owid.catalog.processing as pr
+import pandas as pd
+from owid.catalog import Table
+
+from etl.data_helpers import geo
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+REGIONS = {
+    # Default continents.
+    "Africa",
+    "Asia",
+    "Europe",
+    "North America",
+    "Oceania",
+    "South America",
+}
+
+INCOME_GROUPS = {  # Income groups.
+    "Low-income countries",
+    "Upper-middle-income countries",
+    "Lower-middle-income countries",
+    "High-income countries",
+}
+
+
+def run(dest_dir: str) -> None:
+    # Load datasets.
+    ds_data = paths.load_dataset("gender_statistics")
+    tb = ds_data["gender_statistics"].reset_index()
+    tb_data = tb[["country", "year", "se_prm_uner"]]  # Interested columns.
+
+    ds_regions = paths.load_dataset("regions")
+    ds_income_groups = paths.load_dataset("income_groups")
+    ds_population = paths.load_dataset("population")
+    tb_population = ds_population["population"].reset_index()
+
+    # Clean data to include only years with data.
```

Comment:
You can group these speficif steps in a function for more clarity in the main run function